### DISCUSSION TITLE
Upload backup to a cloud storage

### DISCUSSION THREAD

**Discussion by Genchou** [mbround18/valheim-docker#145] [Ideas]
Heyo,

Just wondered if there was any work in progress towards allowing to move backups to another storage type than the local filesystem. I know that in my case I would not have a rollback solution if my server's drive was to fail, so I've taken the habit to upload the backup archives somewhere else, but it could obviously be automated.

I would be open to try my hand at it by adding a simple aws upload option to the backup routine, if no one is tackling it yet.

**Comment by mbround18** [comment]
Its best to pair this docker with another container that can handle this for you <3 

For example, I use https://github.com/istepanov/docker-backup-to-s3 and upload to Backblaze B2 :)

**Comment by SircasticFox** [comment]
I use restic in a cronjob every night to backup my mapped saves folder to Backblaze B2.
https://restic.net/
